src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -902,6 +919,47 @@ export class BeneficiariesService {
 		// return this.hasuraService.delete(this.table, { id: +id });
 	}
 
+	public async deactivateDuplicateAG(AadhaarNo: string, exceptId: number) {
```

Comment:
we should add this action into activity log as well

Reply:
Done